src/components/executive-education-2u/EnrollmentCompletedSummaryCard.test.jsx

```diff
@@ -1,40 +1,40 @@
-import { render, screen } from '@testing-library/react';
+import { screen } from '@testing-library/react';
 import '@testing-library/jest-dom';
 import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { useParams } from 'react-router-dom';
 import EnrollmentCompletedSummaryCard from './components/EnrollmentCompletedSummaryCard';
+import { renderWithRouterProvider } from '../../utils/tests';
+
+jest.mock('@edx/frontend-platform', () => ({
+  getConfig: jest.fn(() => ({
+    GETSMARTER_STUDENT_TC_URL: 'https://test.org/terms',
+  })),
+}));
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useParams: jest.fn(),
+}
+));
 
 describe('EnrollmentCompletedSummaryCard', () => {
-  const mockProps = {
-    isCourseAssigned: true,
-    externalDashboardUrl: 'https://test.org/external-dashboard',
-    dashboardUrl: 'https://test.org/dashboard',
-    getStudnetTCUrl: 'https://test.org/terms',
-  };
+  beforeEach(() => {
+    jest.clearAllMocks();
+    useParams.mockReturnValue({ enterpriseSlug: 'test-enterprise-slug' });
+  });
 
   it('renders the card with correct content', () => {
-    render(<IntlProvider locale="en"><EnrollmentCompletedSummaryCard {...mockProps} /></IntlProvider>);
+    renderWithRouterProvider(<IntlProvider locale="en"><EnrollmentCompletedSummaryCard /></IntlProvider>);
```

Comment:
If using `renderWithRouterProvider`, it shouldn't be necessary to mock `useParams` above. Define the route path and `initialEntries` to pass as options to `renderWithRouterProvider`.